from func import *

try:
    print_rec(3, 4)
except Exception as e:
    print(e)
